packages/samples/stencil/src/pages/hsp.html

```diff
@@ -0,0 +1,68 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Atomic in Stencil</title>
+    <link rel="stylesheet" href="/build/atomic/themes/coveo.css" />
+    <script type="module" src="/build/app.esm.js"></script>
```

Comment:
Switch for `https://static.cloud.coveo.com/atomic/v2/atomic.esm.js` to see it fail